docs/user-guide/work-packages/copy-move-delete/README.md

```diff
@@ -8,23 +8,29 @@ keywords: copy work package, delete work package, move work package
 
 # Copy, change project or delete a work package
 
-If you click with the right mouse button in the line of the work package, a list with different editing options for the selected work package opens. Here you can copy the work package, delete it and move it to another project. You can access these features from the work package table or from within a work package.
+If you right  click in a work package table, a list with different editing options for the selected work package will open. Here you can change project, copy the work package, or copy its URL to the clipboard, delete it or move it to another project. 
```

Comment:
Suggestions: 

- "right-click" 
- "editing options will be displayed" (without repeating "for the selected work package")
- "Here,"